vm-superio/src/serial.rs

```diff
@@ -11,6 +11,8 @@
 //! This is done by emulating an UART serial port.
 
 use std::collections::VecDeque;
+use std::error::Error as StdError;
```

Comment:
I would prefer using the full path e.g. `impl<E: std::error::Error> std::error::Error for Error<E> {}`.

Reply:
Using the full path causes a redefinition error of `Error` the suggested change was to use `as` to change the binding name of the import.